overlord/fdestate/fdestate.go

```diff
@@ -155,6 +155,12 @@ type FdeState struct {
 	// PrimaryKeys are the keys on the system. Key with ID 0 is
 	// reserved for snapd and is populated on first boot. Other
 	// IDs are for externally managed keys.
+	// If key 0 is not present, we are on a legacy system with that
+	// does not have a primary key in on of those cases:
```

Comment:
// If key 0 is not present, we are on a legacy system that
// does not have a primary key available, so in one of these cases:

?

Reply:
"does not have a primary key. We are then in one of these cases:"